bitbots_bringup/launch/base.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+    <!-- a bit hacky as we can not have an empty string as default value for an environment variable -->
+    <let name="tf_prefix" value="$(eval '\'$(env ROS_NAMESPACE not_defined)\' if \'$(env ROS_NAMESPACE not_defined)\' != \'not_defined\' else \'\'')"/>
+    <arg name="sim" default="false"/>
+
+    <!-- global parameters -->
+    <node name="parameter_blackboard" pkg="demo_nodes_cpp" exec="parameter_blackboard" args="--ros-args --log-level WARN">
```

Comment:
What does `--ros-args` do in the `args` tag?

Reply:
All args are passed as command line arguments to the executable and are defined by the user. But ros nodes feature a few cli arguments that are exposed by ros (like the log level). To avoid namespace issues, the ros arguments are prefixed with `--ros-args`.

Reply:
All args are passed as command line arguments to the executable and are defined by the user. But ros nodes feature a few cli arguments that are exposed by ros (like the log level). To avoid namespace issues, the ros arguments are prefixed with `--ros-args`.